### PR TITLE
Update from skeleton-generic

### LINKED ISSUE #3: Update from skeleton-generic
This project is still using TravisCI, whereas all of our newer work is using GitHub Actions.  Update this project so that it takes advantage of all of the latest goodies from our [`skeleton-generic`](https://github.com/cisagov/skeleton-generic) project.

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
Pull in the latest and greatest from [`skeleton-generic`](https://github.com/cisagov/skeleton-generic).

Resolves #3.

## 💭 Motivation and Context
This project is pretty far behind (e.g. still using TravisCI, instead of GitHub Actions), so now is a great time to get it current.

## 🧪 Testing
`pytest` runs without any failures and `docker-compose up` successfully spins up a local PCA GoPhish environment.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Already being linted by pretty-format-json
+*.json
+# Already being linted by pretty-format-json
+*.json
```
Comment (mcdonnnj) [inline .prettierignore]: One too many.

**Review by mcdonnnj — APPROVED** [review]
:suspect:

**Review by jsf9k — APPROVED** [review]
LGTM, modulo @mcdonnnj's one comment.

File: .prettierignore (hunk 1 above)
Comment (dav3r) [reply .prettierignore]: Ha, thanks for noticing that @mcdonnnj! Got a little bit merge-happy there. 👾  Cleaned up in https://github.com/cisagov/pca-gophish-composition/pull/5/commits/b42e4582853a09470241d08380e97d194a9a5bd6.

**Pull request merged**